packages/styles/src/components/form-validation.scss

```
@@ -31,9 +31,9 @@
 
   &::before {
     content: '';
+    flex-shrink: 0;
```

While `flex-shrink` generally solves the issue, I think it would be a good idea to add `mask-repeat: no-repeat;` as well for stability. Also, the icon should be aligned at the top, so `align-self: flex-start;`.

```suggestion
    flex-shrink: 0;
    mask-repeat: no-repeat;
    align-self: flex-start;
```